analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2017 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.analyzer.integration
+
+import com.here.ort.analyzer.ManagedProjectFiles
+import com.here.ort.analyzer.PackageManager
+import com.here.ort.downloader.Main
+import com.here.ort.downloader.VersionControlSystem
+import com.here.ort.model.Package
+import com.here.ort.utils.Expensive
+
+import io.kotlintest.Spec
+import io.kotlintest.matchers.beEmpty
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.matchers.shouldNot
+import io.kotlintest.matchers.shouldNotBe
+import io.kotlintest.specs.StringSpec
+
+import java.io.File
+
+abstract class AbstractIntegrationSpec : StringSpec() {
+
```

Comment:
Nit: I thought you adapted to my style ;-)

Reply:
Only for functions but not for classes, it seems ;)